### PR TITLE
Host app on Heroku

### PR DESCRIPTION
**Pull request by jimmykimani**
**What does this PR do?** 
Set up Production and Staging

- staging for the develop branch
- prod for the master branch

- A push to master should trigger deployment to production and develop to staging

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,30 +1,65 @@
...
+django-recaptcha==1.3.0
+django-sortedm2m==1.4.0
+django-tastypie==0.13.3
+djangorestframework==3.2.5
+docutils==0.13.1
+easy-thumbnails==2.4.1
+enum34==1.1.6
+flake8==3.3.0
+gunicorn==19.7.1
+html5lib==0.999999999
```
Comment (egichuri) [inline requirements.txt]: Are all these additional requirements related to deployment?

File: wger/settings_global.py [hunk 2]
```diff
@@ -85,7 +88,9 @@
     'corsheaders',
 
     # django-bower for installing bower packages
-    'djangobower', )
+
+'djangobower', )
+
```
Comment (egichuri) [inline wger/settings_global.py]: Looks like this line's indentation is off

**Review by egichuri — CHANGES_REQUESTED** [review]
Just some minor corrections. Otherwise this looks great

File: requirements.txt (hunk 1 above)
Comment (jimmykimani) [reply requirements.txt]: @egichuri Actually only gunicorn the rest are not

**Pull request merged**